liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/applications/DeployMojoSupport.java

```diff
@@ -315,8 +315,7 @@ protected void verifyAppStarted(String appFile) throws MojoExecutionException {
                     Map<String, File> libertyDirPropertyFiles = getLibertyDirectoryPropertyFiles();
                     CommonLogger logger = new CommonLogger(getLog());
                     setLog(logger.getLog());
-                    getServerConfigDocument(logger, serverXML, configDirectory,
-                            bootstrapPropertiesFile, combinedBootstrapProperties, serverEnvFile, false, libertyDirPropertyFiles);
+                    scd = getServerConfigDocument(logger, libertyDirPropertyFiles);
```

Comment:
Do not need to get serverXML file at line 312 anymore.

Reply:
Done